ui/src/components/Footer.js

```diff
@@ -0,0 +1,27 @@
+import React, {Component} from 'react';
+import {FormattedMessage, defineMessages, injectIntl} from 'react-intl';
+
+const messages = defineMessages({
+  disclaimerText: {
+    id: 'Footer.disclaimerText',
+    defaultMessage: ('PLEASE NOTE: all data entered is available to the public '
```

Comment:
I believe "all" should be capitalized (as in the current production)?

Reply:
Ok, done. English in general doesn't have a grammar rule about that afaik, but most style guides seem to prefer capitalizing it in this case.